### PR TITLE
Address some feedback on the Python guide

### PR DESCRIPTION
**Pull request by brcrista**
### Why:

Closes [issue link]

### What's being changed:

Some feedback from @Steve-Glass:
- Add a link to external docs on `requirements.txt`
- Don't fail runs on linting

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by Steve-Glass** [comment]
@brcrista Thanks for opening this!

**Review by kyinfive — DISMISSED** [review]

**Comment by ramyaparimi** [comment]
@brcrista 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/actions/guides/building-and-testing-python.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ steps:
 
 ### Requirements file
 
-After you update `pip`, a typical next step is to install dependencies from *requirements.txt*.
+After you update `pip`, a typical next step is to install dependencies from [requirements.txt](https://pip.pypa.io/en/stable/cli/pip_install/#example-requirements-file).
```
Comment (stevecat) [inline content/actions/guides/building-and-testing-python.md:221]:
```suggestion
After you update `pip`, a typical next step is to install dependencies from *requirements.txt*. For more information, see [pip](https://pip.pypa.io/en/stable/cli/pip_install/#example-requirements-file).
```

**Review by guntrip — DISMISSED** [review]
Thanks for making this change, @brcrista! It looks great ✨ 

I've made one tiny tweak to the `pip.pypa.io` link. I'll apply that suggested change now and get these changes merged.

**Review by guntrip — APPROVED** [review]

**Pull request merged**